helm/reana/templates/NOTES.txt

```diff
@@ -1,22 +1,20 @@
+The REANA system has been installed!
 
-The REANA system has been installed:
+If you are deploying REANA for the first time, there are a few steps left to
+finalise its configuration.
 
-If you are installing REANA for the first time, there are a few steps left to finalise its configuration.
+1. Initialise the database:
 
-1. Get the REANA-Server pod name:
+    $ kubectl exec deployment/reana-server -- ./scripts/create-database.sh
 
-    $ REANA_SERVER=$(kubectl get pod -l "app={{ .Release.Name }}-server" -o name -o jsonpath='{.items[0].metadata.name}')
+2. Create administrator user and corresponding access token:
 
-2. Initialise the database:
+    $ mytoken=$(kubectl exec deployment/reana-server -- flask reana-admin \
+        create-admin-user --email john.doe@example.org --password mypass1234)
```

Comment:
BTW this command outputs also the following warning:

```
/usr/local/lib/python3.8/site-packages/flask_sqlalchemy/__init__.py:833: FSADeprecationWarning: 
SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default 
in the future.  Set it to True or False to suppress this warning.
 warnings.warn(FSADeprecationWarning(
```

We may want to turn it off?!

Reply:
Added to https://github.com/reanahub/reana-server/pull/278